cmd/repl/repl.go

```diff
@@ -119,6 +136,31 @@ func main() {
 		if isTable && (table.NumLines() > 0 || !tableRendered) {
 			table.Render()
 		}
-		fmt.Println("")
+	}
+
+	if flagPassed("execute", "e") {
+		sqlRun(*cliStmt)
+	} else {
+		sqlFile := os.Stdin
+		if flagPassed("file", "f") {
+			sqlFile, err = os.Open(*sqlFileName)
+			if err != nil {
+				log.Fatal(err)
+			}
+		}
+		defer sqlFile.Close()
```

Comment:
I don't think we need to `Close()` the `os.Stdin`.

Reply:
Done.